### ISSUE TITLE
[Command Prompt/PowerShell]: Screen reader is announcing last line first then announcing whole command when using arrow keys to access previous command.

### ISSUE THREAD

**Issue by ghost** [microsoft/terminal#14204]
### Windows Terminal version

1.16.2642.0

### Windows build number

10.0.25217.1000

### Other Software

OS: Window 11 22H2 OS Build 25179.1000
Screen Reader: Narrator
App: Window Terminal

### Steps to reproduce

**Repro Steps:**
Step 1: Open 'PowerShell' and start Narrator.
Step 2: Use 'cd D:\xperf\latest' commands to set the path.
Step 3: Use some commands link 'xperf -merge and .\xperf -help processing (.\xperf -i)'.
Step 4: Now use up arrow keys to access (Merge) previous command.
Step 4: Observe the screen reader behavior.

**User Experience:**
People with visual impairment who use Screen reader on terminal app will be impacted here as command is announced to them in wrong order.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

**Attachments:**
[Repro Video.zip](https://github.com/microsoft/terminal/files/9775837/Repro.Video.zip)

### Expected Behavior

Screen reader should announce only announced changed command text when using arrow keys to access previous command.

Ex: It should announce as '.\xperf<break time="250"/>-merge C:\Users\v-asimeena\Documents\WPR Files\DESKTOP-J6E4AKP.02-25-2022.00-27-09.etl, C:\Users\v-asimeena\Documents\WPR Files\DESKTOP-J6E4AK, P.02-25-2022.00-27-56.etl, C:\Users\v-asimeena\Documents\WPR Files\Test.etl'.

### Actual Behavior

Screen reader is announcing last line first then announcing whole command when using arrow keys to access previous command.

Same issue repro with every command which is more than 1 line.

i.e. Screen Reader is announcing as 'P.02-25-2022.00-27-56.etl C:\Users\v-asimeena\Documents\WPR Files\Test.etl, .\xperf<break time="250"/>-merge C:\Users\v-asimeena\Documents\WPR Files\DESKTOP-J6E4AKP.02-25-2022.00-27-09.etl, C:\Users\v-asimeena\Documents\WPR Files\DESKTOP-J6E4AK, P.02-25-2022.00-27-56.etl, C:\Users\v-asimeena\Documents\WPR Files\Test.etl'.

**Comment by v-rpundir** [comment]
Dependent Bug: [Bug 38332898](https://microsoft.visualstudio.com/OS/_workitems/edit/38332898): [XPerf CMD]: Screen reader is not announcing descriptively when using arrow keys to access previous (Merge) command.

**Comment by carlos-zamora** [comment]
Verified this issue still repros on 1.18.1462.0.
This issue also repros on PowerShell and CMD. I even tried removing the PSReadline module in PowerShell and that still doesn't fix the issue.

**Comment by Priyanka-Chauhan123** [comment]
Fixed in V1.17.11461.0. Hence, closing the bug
